float64 trans_vel
float64 trans_tolerance
float64 rot_vel
float64 time_out
geometry_msgs/PoseStamped goal
---
geometry_msgs/Vector3 error
string message
---
geometry_msgs/Vector3 error
